app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -263,6 +280,14 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
             isEnabled = syncEnginesStatus.containsKey(SyncEngine.Passwords)
             isChecked = syncEnginesStatus.getOrElse(SyncEngine.Passwords) { true }
         }
+        val tabsNameKey = getPreferenceKey(R.string.pref_key_sync_tabs)
+        findPreference<CheckBoxPreference>(tabsNameKey)?.apply {
+
+            this.isVisible = shouldShowSyncTabs
+
+            isEnabled = syncEnginesStatus.containsKey(SyncEngine.Tabs)
+            isChecked = syncEnginesStatus.getOrElse(SyncEngine.Tabs) { true }
```

Comment:
We probably want the default to be false here since we're hiding the setting from some users for now